client/styles/styles.css

```diff
@@ -372,3 +372,27 @@ body > div:first-of-type > div:first-of-type {
   }
 }
 
+/*----------------- requester order -----------------*/
+
+#requester-order-details, #requester-order-complete, #requester-order-rate {
+  margin-left: 10px;
+}
+
+.order-details {
+  border-style: solid;
```

Comment:
I use the `border: 1px solid #ffffff` style. So we may want to match it.
It combines `border-width`,`border-style`,and `border-color` into one